### DISCUSSION TITLE
Player Defuse Name incorrect?

### DISCUSSION THREAD

**Discussion by NG-Clark** [redraskal/r6-dissect#112] [Q&A]
I got everything running great for my team, but it seems the username for the Defuse is incorrect. In round two it was A player called HydroTahiti.UVU that planted the defuser, however it is showing the username as Solo.UU. Is this a bug? For reference, Solo.UU never touched defuser the whole game. Everything else is correct however.

{"type":{"name":"DefuserPlantComplete","id":3},"username":"Solo.UU","time":"0:47","timeInSeconds":47},

**Comment by redraskal** [comment]
yeah this is a bug with defuse tracking
